src/codegen/sys/lib_.rs

```diff
@@ -389,7 +397,8 @@ fn generate_interfaces_structs(
         if !env.type_status_sys(&full_name).need_generate() {
             continue;
         }
-        writeln!(w, "#[repr(C)]\npub struct {}(c_void);\n", interface.c_type)?;
+        writeln!(w, "#[repr(C)]")?;
```

Comment:
Why not move the `#[repr(C)]` into the `generate_opaque_type` function?

Reply:
I did, but that creates quite the diff haha.

Reply:
Why?

Reply:
I see :man_shrugging: